[2025-03-21T21:00:03.603+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-21T21:00:03.612+0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_bluesky.ingest_bluesky scheduled__2025-03-21T15:00:00+00:00 [queued]>
[2025-03-21T21:00:03.619+0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_bluesky.ingest_bluesky scheduled__2025-03-21T15:00:00+00:00 [queued]>
[2025-03-21T21:00:03.619+0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-21T21:00:03.641+0500] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): ingest_bluesky> on 2025-03-21 15:00:00+00:00
[2025-03-21T21:00:03.646+0500] {standard_task_runner.py:72} INFO - Started process 104249 to run task
[2025-03-21T21:00:03.650+0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_bluesky', 'ingest_bluesky', 'scheduled__2025-03-21T15:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/fetch_bluesky.py', '--cfg-path', '/tmp/tmpj96h4pig']
[2025-03-21T21:00:03.651+0500] {standard_task_runner.py:105} INFO - Job 28: Subtask ingest_bluesky
[2025-03-21T21:00:03.691+0500] {task_command.py:467} INFO - Running <TaskInstance: fetch_bluesky.ingest_bluesky scheduled__2025-03-21T15:00:00+00:00 [running]> on host ubuntu
[2025-03-21T21:00:03.760+0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fetch_bluesky' AIRFLOW_CTX_TASK_ID='ingest_bluesky' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-21T15:00:00+00:00'
[2025-03-21T21:00:03.763+0500] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-21T21:00:03.763+0500] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-21T21:00:03.764+0500] {logging_mixin.py:190} INFO - Current task name:ingest_bluesky state:running start_date:2025-03-21 16:00:03.613310+00:00
[2025-03-21T21:00:03.764+0500] {logging_mixin.py:190} INFO - Dag name:fetch_bluesky and current dag run status:running
[2025-03-21T21:00:03.764+0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-21T21:00:05.099+0500] {logging_mixin.py:190} INFO - Failed to authenticate. Status code: 404
[2025-03-21T21:00:05.099+0500] {logging_mixin.py:190} INFO - Authentication failed, aborting data fetch.
[2025-03-21T21:00:05.100+0500] {python.py:240} INFO - Done. Returned value was: None
[2025-03-21T21:00:05.108+0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-21T21:00:05.109+0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fetch_bluesky, task_id=ingest_bluesky, run_id=scheduled__2025-03-21T15:00:00+00:00, execution_date=20250321T150000, start_date=20250321T160003, end_date=20250321T160005
[2025-03-21T21:00:05.132+0500] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-21T21:00:05.133+0500] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-21T21:00:05.133+0500] {logging_mixin.py:190} INFO - Dag name:fetch_bluesky queued_at:2025-03-21 16:00:01.193573+00:00
[2025-03-21T21:00:05.133+0500] {logging_mixin.py:190} INFO - Task hostname:ubuntu operator:PythonOperator
[2025-03-21T21:00:05.146+0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-21T21:00:05.176+0500] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-21T21:00:05.177+0500] {local_task_job_runner.py:245} INFO - ::endgroup::
